### ISSUE TITLE
Bangladesh, Kenya, India: Produce metrics and pie charts

### ISSUE THREAD

**Issue by LauraReynal** [MozillaFoundation/research#21]
*Goal*: take data from all past research and combine the useful metrics into pie charts for a future synthesis report. 

Phase: Research 
Owner: @LauraReynal
in collaboration with: @adamlofting 
Lead design: n/a
Lead dev: n/a
Quality: @thisandagain, @secretrobotron 

To do: 
- [x] make a list of the existing metrics and choose the relevant ones
- [x] submit to team for quality
- [x] Adam set up the tables
- [ ] Laura goes through Bangladesh
- [ ] Laura goes through Kenya
- [x] Laura goes through India
- [ ] @adamlofting makes pie charts 
- [ ] Choosing relevant charts and conclude

**Comment by LauraReynal** [comment]
CC / @secretrobotron @thisandagain @benrito @cecilepompei @davidascher 
We identified existing metrics from the research, some are already quantified and 'easily' ready to be used, and some will take us time to go through all content again. Feel free to add / withdraw your questions ! 
Cheers 
----
First list of metrics identified 

**Non - tech**
- Age => make age groups
- Gender 
- Evironment => urban  - rural - town - urban slum 
- Occupation => To do : come up with a list of categories. do we make it simple or represent our diversity?
- Education level => go through alll interviews  // come up with classification 
- English speaking level => come up with classification that makes sense 
- Literacy level => is able to read and write 
- Digital literacy level => Classify each participant in the web journey categories => Add a column in the spreadsheets \
- How many of the youth (teen to university students ) are hustling for money and have side businesses // where country are they from => ""In kenya 79% of students interviewed have their own micro business, whereas in Bangladesh, only 5%""

------

**tech**
- number of mobile phones using 
- brand and model of phones 
  - => make a categorization of phones 
  - => brands 
  - => types of OS
- Devices 
  - number of devices used => 
  - types of devices => laptops, tablets, smartphones etc 
  - only have a smartphone 
- Accessing the Web =>  How many of the people who access the web only do it through their mobile ?
  - Cyber cafe access / Wifi in office or campus / Access at home 
==> People accessing the web trhough mobile devices and also through cyber cafe because of mobile constraints or data cost. 
- Use of social media = yes or no 
- preferred social media 
- use of social media for business 
- Sells with the Web (social media or other ecommerce, regular use or not ) 
- How many people engage with local content or services regularly ? (to do= define the frequency, define the "local content") 
- what is the primary language of Web consumption for people ? English / hindi / etc 
- Favorite indian/bangladeshi/kenyan websites 
- Spending on mobile per month / per country / per gender / per occupation => make a classification and convert all currency  // couple this with cost per country (ask the GSMA or other ) 
- How many people perceive the Web as a way to make money ? => Before Laura brought it up ( x% of people spontenously mentioned .... ) 


------

**making / sharing / learning and remixing**

- Number of people already creating content on the web when interviewed ? ( Define content creation ie not facebook or instagram... publishing to a whole world wide web as opposed to a constrained social media environment )
- Number of people not aware that you dont need to code to create content 
- Number of people expressing lack of confidence to create contnet (fear etc)
- Number of people expressing lack of interest // **interest** in creating 
- Same with learning 
  - => Couple the above with => gender / age / country / occupation 
- Primary motivators for sharing content => Define the motivators ( connecting with friends, reputation, money, learning )
- % of people who are likely to use webmaker ( purely qualitative estimates) 


** internal and methods ** 
- % of people who tested the app and provided feedback 
- Internal only; classify socio-economic environment ( based on observations and not quantitative data)

**Comment by adamlofting** [comment]
Next week I will spend time formatting the spreadsheets in a way that makes it easier to input this new categorical data, and which also makes the number crunching easier afterwards. 

Then where the data is comparable across countries, produce some standardized reports to write look at total numbers, and differences by country.

**Comment by LauraReynal** [comment]
Did a pass on the spreadsheets Adam ! Let me know if anything unclear :santa: :christmas_tree:

**Comment by cecilepompei** [comment]
Hi there!

I would add in non-tech
- how many people own a mobile phone in your family/household?
- how many smart phone in the household?
- how many digital devices in the household
- total revenue in the household (when people feel confident to share)

I will pull in research I did on micro entrepreneurs and see what is relevant to get here

cheers

**Comment by benrito** [comment]
@adamlofting any update on the table-ification?

**Comment by adamlofting** [comment]
It hasn't happened yet (a couple of things didn't go to plan last week). 

This should be done before @LauraReynal and I are meeting on Thursday morning.

**Comment by adamlofting** [comment]
Hey @cecilepompei - those are great questions which we should include in future research plans, but as far as I know, we don't have answers to those from the existing work. @LauraReynal is that correct?

**@ All : Update on this:**

* I've standardized the spreadsheet headings and categorical results to prepare these for analysis
  * These are consistent across the 3 country spreadsheets now
* There are a few open questions around how to present data when the interview format changed (group interviews for example) @LauraReynal and I will work through these
* I've researched the devices in use to add the OS version info for other webmaker/teach requirements gathering
* I've added new columns as needed to extract the metrics listed above in this issue
  * Some of these I can complete based on the open text responses in place
  * Others will need back-filling by re-processing the original documentation
* Once we finish the data cleaning we can start analyzing the data

**Comment by LauraReynal** [comment]
@adamlofting @cecilepompei 
I might have some of these info, but definetely dont have them all. It will be implemented in future research though. Thanks 


@adamlofting Thanks for the spreadsheet Pass ! I'll answer your comments / questions and take a pass at it. This way thursday we can start the maths :dancer:

**Comment by LauraReynal** [comment]
Spent time with @adamlofting reviewing the tables. 
next step : Fill out all empty columns - Which means I have to go through all my interview notes. 
TL DR : @benrito this is going to take some time ! Will keep you posted .

**Comment by LauraReynal** [comment]
cc / @corinagardner & @APLSmith FYI work in progress on the metrics

**Comment by benrito** [comment]
all: can you share the document link?

**Comment by LauraReynal** [comment]
There is no doc yet

**Comment by LauraReynal** [comment]
@ALPSmith I have completed / updated the bangladesh Spreadsheet with the notes

**Comment by APLSmith** [comment]
Great - thanks.  Reviewing now.

**Comment by LauraReynal** [comment]
@adamlofting @APLSmith 
I have filled out all the India table. 
Im waiting for @APLSmith to answer lignes 63 and 64 - but you should be able to start having fun. 
Bangladesh is 40% done and Kenya 0%

**Comment by APLSmith** [comment]
@adamlofting @LauraReynal  
Have done lines 63 and 64

**Comment by LauraReynal** [comment]
No longer needed by the team. Closing
